### ISSUE TITLE
crono docker container fails to start

### ISSUE THREAD

**Issue by lkoenigsberger** [SUSE/Portus#507]
I get the following error when I try to start the crono container over docker-compose:
```
28.10.2015 12:35:37/usr/lib/python2.7/dist-packages/supervisor/options.py:296: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
28.10.2015 12:35:37  'Supervisord is running as root and it is searching '
28.10.2015 12:36:24/usr/lib/python2.7/dist-packages/supervisor/options.py:296: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
28.10.2015 12:36:24  'Supervisord is running as root and it is searching '
28.10.2015 12:36:24Unlinking stale socket /var/run/supervisor.sock
```

**Comment by mssola** [comment]
This might be fixed with PR #634 too, but I'm not sure.

**Comment by flavio** [comment]
I think we can close the issue right now, we got rid of supervisord so the error message (if still present) is going to be completely different. Feel free to open a new issue if needed.
